react/features/base/participants/reducer.js

```diff
@@ -59,9 +61,11 @@ const DEFAULT_STATE = {
     fakeParticipants: new Map(),
     haveParticipantWithScreenSharingFeature: false,
     local: undefined,
+    localScreenShare: undefined,
```

Comment:
I wanted to create a separate state for the local SS participant to decouple it from sortedFakeScreenShareParticipants. This is helpful when we need logic that needs to know if the presenter is screen sharing